### ISSUE TITLE
[high] JAWS+Chrome: Many Pages: Checkbox announced without role, state and label

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5648]
## Action Performed:
Prerequisite: the user is logged in.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Navigate to Settings > Profile overlay.
3. Once the Profile overlay opens, press Tab key to focus the checkbox "Set my timezone automatically".
4. Note how the element is announced.

## Expected Result:
The checkbox should be announced with a checkbox role, checked/not checked state and associated label.

## Actual Result:
The checkbox announces with no label, role or state and also has multiple focus stops.

HTML:
`<div class="css-1dbjc4n r-1awozwy" style="flex-direction: row; -webkit-box-orient: horizontal; -webkit-box-direction: normal;"><div tabindex="0" class="css-1dbjc4n r-1loqt21 r-1otgn73"><div class="css-1dbjc4n r-1awozwy r-z80fyv r-1777fci r-19wmn03" style="background-color: rgb(1, 133, 255); border-color: rgb(236, 236, 236); border-radius: 2px; border-width: 1px;"><svg xmlns="http://www.w3.org/2000/svg"
viewBox="0 0 20 20" width="14" height="14" fill="white"><path d="M17.1 5.5l-.9-.9c-.2-.2-.6-.2-.9 0l-7.6 7.6-3.1-3c-.2-.2-.6-.2-.8 0l-.9.9c-.2.2-.2.6 0 .9L6 14.1l1.4 1.4c.2.2.6.2.9 0l1.4-1.4 7.6-7.6c.1-.4.1-.7-.2-1z"></path></svg></div></div><div tabindex="0" class="css-1dbjc4n r-1awozwy r-1loqt21 r-1otgn73 r-1i6wzkk r-lrvibr r-13qz1uu" style="flex-flow: row wrap; margin-left: 8px; padding-right: 8px; -webkit-box-orient: horizontal; -webkit-box-direction: normal; transition-duration: 0.25s;"><div dir="auto" class="css-901oao" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; margin-left: 8px;">Set my timezone automatically</div></div></div>`

Other occurrences:
1. On the New Group overlay, a similar issue is present with the custom checkboxes used to select participants in the group: the checkboxes announce no role or state.

## Workaround:
No workaround

## Area issue was found in:
Profile overlay, Create new group overlay

## Failed WCAG checkpoints
4.1.2

## User impact:
The screen reader user might not understand what element is focused.

## Suggested resolution:
Review the current implementation and ensure that the checkbox is implemented correctly, so that the role, state and label announces upon focusing the element.
https://webaim.org/techniques/forms/controls#checkbox
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/136039684-3eb06954-635a-43aa-8190-2c39c3c92c56.mp4

![Uploading Bug5184932_info_and_relationship_-_checkbox_-_profile.png…]()

https://user-images.githubusercontent.com/88733897/136039723-7ad0026b-3741-439c-95b5-1d957059c0cd.mp4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
